redux/redux.d.ts

```diff
@@ -1,52 +1,81 @@
-// Type definitions for Redux v1.0.0
-// Project: https://github.com/rackt/redux
-// Definitions by: William Buchwalter <https://github.com/wbuchwalter/>, Vincent Prouillet <https://github.com/Keats/>
```

Comment:
I can add both original authors, no problem. I replaced them in this file because I build all definitions from a scratch. They still appear in `v1.0.0` legacy version.